wled00/FX_fcn.cpp

```diff
@@ -1152,4 +1152,5 @@ uint32_t WS2812FX::gamma32(uint32_t color)
 }
 
 WS2812FX* WS2812FX::instance = nullptr;
-int16_t Bus::_cct = -1;
\ No newline at end of file
+int16_t Bus::_cct = -1;
```

Comment:
I do not like this at this place.
Perhaps moving it to util.cpp at the beginning or converting bus_manager.h into .cpp & .h combo.

Reply:
Agreed, me neither. Should really make a `bus_manager.cpp`, though I dislike it for two reasons. a) Too many files and b) always have to change/add functions in two places.